build.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
Do you find it helpful to have a build script over just having the build commands documented in the README?  I know I will not use it myself as I will need more complex build commands as I have to use buildx for multi-arch building and I also the buildx cache.

Reply:
You are right, and actually I hesitated. I will remove `build.sh`